### ISSUE TITLE
React 17+ peer dependency?

### ISSUE THREAD

**Issue by jayqui** [CognizantStudio/react-loader#84]
Hi there, on a project that uses `react-loader 2.4.7`, I'm getting a peer dependency warning:

```
warning " > react-loader@2.4.7" has incorrect peer dependency "react@^0.14.0 || ^15.0.0 || ^16.0.0".
```

That must be because of the `peerDependencies` [specified](https://github.com/CognizantStudio/react-loader/blob/master/package.json#L37) in this project's `package.json`.

Could this be as simple as appending a `|| ^17.0.0"` to that line of code? One reason to think so is that React 17 essentially adds [no new features](https://reactjs.org/blog/2020/10/20/react-v17.html#no-new-features).

Thanks for any input!

**Comment by jrhunter84** [comment]
Any changes on this? I too am upgrading to 17.0.2 and I'm getting a dependency error when trying to upgrade.

**Comment by jayqui** [comment]
Seems like this project isn't being very actively maintained. I think I'm going to have us just switch to a different spinner. After all, it would be simpler to rely on a dependency in which you don't have to wrap your component inside a `<Loader />` higher-order component but rather simply do an early `return <SomeSpinner />`.

**Comment by jrhunter84** [comment]
I ended up swiching to Material-UI's Backdrop. I was already using other components of mui so I didn't have to look for another package, and it works like the second scenario you mentioned (no wrapping). Hope this helps you!

**Comment by nicholasserra** [comment]
Yup i'm not sure who still has access to this. I doubt it's being monitored. Maybe @mikekauffman knows.
